src/components/styles.js

```diff
@@ -30,6 +30,25 @@ export const globalSyles = {
   },
 };
 
+export const thumbnail = css({
```

Comment:
If these are specific to a component can you add them to the component file?

Reply:
absolutely, didn't know what the logic was there!